src/common/transactions/transaction-utils.ts

```diff
@@ -49,6 +49,32 @@ function txHasTime(tx: Tx) {
   );
 }
 
+const maxValueForLowFeeEstimation = 5000000;
+const maxValueForMiddleFeeEstimation = 15000000;
+const maxValueForHighFeeEstimation = 25000000;
+
+export function getFeeEstimationsWithMaxValues(feeEstimations: FeeEstimation[]) {
+  const feeEstimationsWithMaxValues = [];
+
+  if (new BigNumber(feeEstimations[0].fee).isGreaterThan(maxValueForLowFeeEstimation)) {
+    feeEstimationsWithMaxValues.push({ fee: maxValueForLowFeeEstimation, fee_rate: 0 });
+  } else {
+    feeEstimationsWithMaxValues.push(feeEstimations[0]);
+  }
+  if (new BigNumber(feeEstimations[1].fee).isGreaterThan(maxValueForMiddleFeeEstimation)) {
+    feeEstimationsWithMaxValues.push({ fee: maxValueForMiddleFeeEstimation, fee_rate: 0 });
+  } else {
+    feeEstimationsWithMaxValues.push(feeEstimations[1]);
+  }
+  if (new BigNumber(feeEstimations[2].fee).isGreaterThan(maxValueForHighFeeEstimation)) {
+    feeEstimationsWithMaxValues.push({ fee: maxValueForHighFeeEstimation, fee_rate: 0 });
+  } else {
+    feeEstimationsWithMaxValues.push(feeEstimations[2]);
+  }
```

Comment:
Alternate implementation might look something like this:

```suggestion
const maxHighFeeEstimations = [
  maxValueForLowFeeEstimation,
  maxValueForMiddleFeeEstimation,
  maxValueForHighFeeEstimation,
];

export function getFeeEstimationsWithMaxValues(feeEstimations: FeeEstimation[]) {
  return feeEstimations.map(({ fee, ...rest }, index) => ({
    ...rest,
    fee: BigNumber.min(fee, maxHighFeeEstimations[index]),
  }));
}
```

Mapping an array to get the result, using `BigNumber.min` to select the lower of the two values.

Reply:
Ok, thanks for the suggestion. Will keep in mind for next time.